packages/teams-js/test/public/visualMedia.spec.ts

```diff
@@ -0,0 +1,517 @@
+import { errorLibraryNotInitialized, permissionsAPIsRequiredVersion } from '../../src/internal/constants';
+import { GlobalVars } from '../../src/internal/globalVars';
+import { DOMMessageEvent } from '../../src/internal/interfaces';
+import { visualMedia } from '../../src/public';
+import { app } from '../../src/public/app';
+import { errorNotSupportedOnPlatform, FrameContexts } from '../../src/public/constants';
+import { DevicePermission, ErrorCode } from '../../src/public/interfaces';
+import { setUnitializedRuntime } from '../../src/public/runtime';
+import { Utils } from '../utils';
+
+/* eslint-disable */
+/* As part of enabling eslint on test files, we need to disable eslint checking on the specific files with
+   large numbers of errors. Then, over time, we can fix the errors and reenable eslint on a per file basis. */
+/**
+ * Test cases for visualMedia APIs
+ */
+describe('visualMedia', () => {
+  const originalDefaultPlatformVersion = '1.6.0';
+  const allowedContexts = [FrameContexts.content, FrameContexts.task];
+  const minVersionForPermissionsAPIs = permissionsAPIsRequiredVersion;
+  const errorInvalidCount = 'Invalid input count: Must supply the valid image counts (limit to 10).';
```

Comment:
Move this to constant and import it here.

Reply:
Moved to constant.ts